tests/mongodb/apps/default-app/functions/confirmFunc/source.js

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Why add all these things to "default-app" when it's not used by CI and seems to be redundant with the "integration-tests" app? Is it for manually testing?

Reply:
it was for local testing, I was going around in circles with some of the docker stuff locally